core/src/dunit/scala/io/snappydata/cluster/SplitClusterDUnitTest.scala

```diff
@@ -549,10 +549,10 @@ object SplitClusterDUnitTest extends SplitClusterDUnitTestObject {
 
     // also check access to complex types as string
     rs = stmt.executeQuery(
-      s"SELECT * FROM $tableName --+ complexTypeAsClob(1)")
+      s"SELECT * FROM $tableName --+ complexTypeAsJson(1)")
```

Comment:
The hint is gone for clob which is a super type for JSON. So all complex type can be represented as JSON?